src/javascript/app/common/attach_dom/dialog.js

```diff
@@ -17,16 +17,43 @@ const Dialog = (() => {
                     const el_btn_ok     = container.querySelector('#btn_ok');
                     const el_btn_cancel = container.querySelector('#btn_cancel');
                     const el_title      = container.querySelector('#dialog_title');
+                    const el_footnote   = container.querySelector('#dialog_footnote');
 
                     if (!el_dialog) return;
 
-                    const localized_message = Array.isArray(options.localized_message) ? options.localized_message.join('<p />') : options.localized_message;
-                    elementInnerHtml(container.querySelector('#dialog_message'), localized_message);
+                    // Used for setting the dialog title, message, and footnote
+                    const setMessage = (option_item, $selector) => {
+                        const item = Array.isArray(options[option_item]) ? options[option_item].join('<p />') : options[option_item];
+                        elementInnerHtml(container.querySelector($selector), item);
+                    };
 
-                    const localized_title = options.localized_title;
-                    if (localized_title && el_title) {
+                    // Set title if `localized_title` field in `options` param exists
+                    if (options.localized_title && el_title) {
```

Comment:
What about this instead of the comment? 
```suggestion
                    const has_title = options.localized_title && el_title
                    if (has_title) {
```